deployment-configuration.md

```diff
@@ -37,7 +39,20 @@ and Unicorn on Ubuntu 16.
 
 2. Setup the crucible_smart_app.
 
-Issue the following commands to setup the crucible smart app.
+To use all features, install Ruby 2.5. Enter the following commands in your terminal.
+
+
+    sudo apt-get install libgdbm-dev libncurses5-dev automake libtool bison libffi-dev
+    gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+    curl -sSL https://get.rvm.io | bash -s stable
+    source ~/.rvm/scripts/rvm
+    rvm install 2.5.0
```

Comment:
Can we do ```rvm install 2.5``` instead, to just do the most recent patch on 2.5 (does rvm work like this?)

Reply:
the `.0` is required.  the `2.5` failed for me.